deploy/centos7x64-pgsql.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+sudo rpm -Uvh https://yum.postgresql.org/11/redhat/rhel-7-x86_64/pgdg-centos11-11-2.noarch.rpm
+sudo yum install -y postgresql11 postgresql11-server postgresql11-contrib
+sudo /usr/pgsql-11/bin/postgresql-11-setup initdb
+sudo systemctl enable postgresql-11
+sudo systemctl start postgresql-11
```

Comment:
```suggestion
sudo systemctl enable --now postgresql-11
```
For both enable and start.